README.md

```diff
@@ -95,6 +93,14 @@ Per-VDOM:
  * `fortigate_virtual_wan_bandwidth_rx_byte_per_second`
  * `fortigate_virtual_wan_status_change_time_seconds`
 
+ Per-BGP-Neighbor and VDOM:
+ * `fortigate_bgp_neighbor_ipv4_info`
+ * `fortigate_bgp_neighbor_ipv6_info`
+ * `fortigate_bgp_neighbor_paths_ipv4_count`
```

Comment:
The `_count` suffix is sort of reserved for Prometheus counters, and this is a  Gauge. So maybe call these `fortigate_bgp_neighbor_ipv4_paths` and `fortigate_bgp_neighbor_ipv4_best_paths`?